docs/maintainers/adr/0020-reduce-support-for-internet-explorer.md

```diff
@@ -0,0 +1,77 @@
+# Reduce support for Internet Explorer
+
+* Status: accepted
+* Date: 2023-12-19
+
+Technical Story: PLATUI-2686
+
+## Context and Problem Statement
+
+[govuk-frontend v5.0.0](https://github.com/alphagov/govuk-frontend/releases/tag/v5.0.0)
+drops support for Internet Explorer (IE) versions 8 through 10 by preventing loading of its Javascript (JS)
+and removing various IE-specific polyfills.  IE11 is still supported via graceful degradation to a non-JS experience.
+
+We propose to follow the same approach in our frontend libraries, preventing loading of our own JS and
+removing any IE-specific polyfills.
+
+## Decision Drivers
+
+* Direction taken by Government Digital Service
+* Industry trends to drop support for IE
+* Overhead of maintaining and testing IE-specific implementations
+* Minimal number of users still using IE (recent sample found the number was less than 0.6% of users over a 2-week period)
```

Comment:
Should we add in link to Service Manual saying threshold of 2% for support?
https://www.gov.uk/service-manual/technology/designing-for-different-browsers-and-devices

Reply:
good shout - added link for context